testgrid/config/config.yaml

```diff
@@ -2863,8 +2863,62 @@ dashboards:
 
 - name: sig-apps
   dashboard_tab:
-  - name: TODO
-    test_group_name: ci-kubernetes-e2e-gce-slow  # TODO(fejta): fix
+  - name: gce
+    test_group_name: ci-kubernetes-e2e-gce
+    base_options: 'include-filter-by-regex=sig-apps'
+    description: 'apps gce e2e tests for master branch'
+  - name: gke
+    test_group_name: ci-kubernetes-e2e-gke
+    base_options: 'include-filter-by-regex=sig-apps'
+    description: 'apps gke e2e tests for master branch'
+  - name: gce-1.7
+    test_group_name: ci-kubernetes-e2e-gce-release-1-7
+    base_options: 'include-filter-by-regex=sig-apps'
```

Comment:
Good point @rmmh. Yeah, the tag sig-apps only exists in the new e2e tests, so, it wouldn't find anything in old releases. They had the volume tag in older releases which is why it worked. I think we should get rid of these two dashboards because it's not worth backporting the addition of "sig-apps" to those branches.